### ISSUE TITLE
Invalid position given by 'E015'

### ISSUE THREAD

**Issue by tyrannosaurus-jamie** [AtomLinter/linter-jshint#481]
JSHint returned a point that did not exist in the document being edited.
Rule: `E015`
Requested point: 6:13

```
class ProductList extends React.Component {

	constructor(props) {
		super(props);

		this.state = {
			products: [],
		};
	}

	handleProductUpVote(productId) {
		console.log(productId + ' was upvoted.');
	}
}
```
Debug information:
```
Atom version: 1.18.0
linter-jshint version: v3.1.5
JSHint version: jshint v2.9.5
Hours since last Atom restart: 0.2
Platform: darwin
Current file's scopes: [
  "source.js",
  "punctuation.definition.function.body.end.bracket.curly.js"
]
linter-jshint configuration: {
  "executablePath": "",
  "lintInlineJavaScript": false,
  "disableWhenNoJshintrcFileInPath": false,
  "scopes": [
    "source.js",
    "source.js-semantic"
  ],
  "jshintFileName": ".jshintrc",
  "jshintignoreFilename": ".jshintignore"
}
```

**Comment by Arcanemagus** [comment]
As your code is indented with tabs this is another case of https://github.com/jshint/jshint/issues/3151. Please follow that issue for when this will be fixed in JSHint.

If anyone is directed here and _isn't_ using tabs to indent your file please file a new issue with a file to reproduce it on the JSHint repo, or file one here and I'll try to help you in getting one filed there where it can be fixed.

**Comment by Arcanemagus** [comment]
There are reports that if you set your `indent` property of your JSHint configuration to `1` this issue can be worked around. If you don't have a `.jshintrc` file the following should work:
```json
{
  "indent": 1
}
```

**Comment by johnnya23** [comment]
@Arcanemagus can you be more specific about this solution

> There are reports that if you set your indent property of your JSHint configuration to 1 this issue can be worked around. If you don't have a .jshintrc file the following should work:
> 
> {
>   "indent": 1
> }
